.github/CONTRIBUTING.md

```diff
@@ -50,11 +57,11 @@ The `Destroy()` proc for objects should be defined, if there are any special ope
 Note that any modified `Destroy()` proc **must always return the original definition (`return ..()`) call!**
 
 An example of how to define `Destroy()` for an item that needs it:
-```
+```dm
```

Comment:
DM v dm
All versions above this are capitals, all below are lowercase. This mildly bugs me